extends Node2D

func _ready() -> void:
	Globals.otk = true
	_translate_otk()

func _translate_otk():
	$basla.text = "ð°‹ð°€ð±ð°žð°€"
	$latin.text = "ð°žð°€ð±ƒð°ƒð°¤"
	$orkun.text = "ð°†ð°ºð°¸ð°£"
	$W.text = "W / ð°‰ð°†ð±ð°žð°¸"
	$E2.text = "ð°€ð±…ð°šð°ƒð° ð°€ð±ð°ƒð°¢"
	$K2.text = "ð°½ð°€ð°¡ð°ƒð°ºð°ƒ"
	$L2.text = "ð°´ð°€ð°ð°£ð°ƒð°–:ð±ƒð°†ð±ƒ"
	
func _translate_latin():
	# basla
	$basla.text = "BaÅŸla"
	$latin.text = "Latin"
	$orkun.text = "Orkun"
	$W.text = "W / BoÅŸluk"
	$E2.text = "EtkileÅŸim"
	$K2.text = "SaldÄ±rÄ±"
	$L2.text = "KaÄŸnÄ±yÄ± Tut"
	


func _on_basla_pressed() -> void:
	get_tree().change_scene_to_file("res://sahneler/aralevel/main.tscn") # Replace with function body.


func _on_latin_pressed() -> void:
	_translate_latin()
	Globals.otk = false


func _on_orkun_pressed() -> void:
	_translate_otk()
	Globals.otk = true
